### PR TITLE
add gmontano671 Guam recommendation file

### LINKED ISSUE #18: Add file for gmontano671's Hometown
- [x] create a branch!
- [x] make commits
- [x] create a pull request
- [ ] collaborate
- [ ] review
- [ ] merge

### PR DESCRIPTION
**Pull request by gmontano671**
review and please add me!

closes #18

### CODE REVIEW

**Comment by fineticlabs** [comment]
🥇

File: gmontano671-guam.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Guam
+
+### Places to Eat
+- Jamaican Grill
+- King's
```
Comment (fineticlabs) [inline gmontano671-guam.md:5]: Really hate this place 👎

File: gmontano671-guam.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Guam
+
+### Places to Eat
+- Jamaican Grill
```
Comment (fineticlabs) [inline gmontano671-guam.md:4]: Looks good 👍

File: gmontano671-guam.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+- King's
+- Capriciossa
+
+### Places to Visit
+- Chamorro Village
```
Comment (fineticlabs) [inline gmontano671-guam.md:9]: 🐻

**Review by fineticlabs — COMMENTED** [review]
GenCom

File: gmontano671-guam.md (hunk 1 above)
Comment (gmontano671) [reply gmontano671-guam.md:5]: There are better spots than King's

File: gmontano671-guam.md [hunk 4]
```diff
@@ -10,3 +10,4 @@
 
 ### Places to Avoid
 - Yona Village
+- Public Bathrooms
```
Comment (fineticlabs) [inline gmontano671-guam.md]: where do I go then?

**Review by fineticlabs — CHANGES_REQUESTED** [review]
- Please elaborate on Bathroom activities.

**Pull request merged**